internal/pkg/cli/dialog/create_template.go

```diff
@@ -185,22 +186,23 @@ func (d *createTmplDialog) askObjectsIds() ([]template.ConfigDef, error) {
 func (d *createTmplDialog) parseObjectsIds(result string) ([]template.ConfigDef, error) {
 	idByKey := make(map[string]string)
 	ids := make(map[string]bool)
-	errors := utils.NewMultiError()
+	result = strhelper.StripHtmlComments(result)
```

Comment:
Nahradime komentare za prazdne riadky, aby sme mali jednoduchsie parsovanie.
Komentare neodstranime uplne, ale nechame prazdne riadky, aby sedelo cislo `line`, ked je v subore nejaka chyba.

Reply:
Markdown podporuje HTML komentare, je to najprehladnejsi sposob ako v MD zapisat komentar.